packages/frontend/src/styles/style.css

```diff
@@ -3,6 +3,7 @@
 @import 'tailwindcss/utilities';
 
 @import './markdown.css';
+@import './article.css';
```

Comment:
What is the difference between markdown and article? Can we have a single file that styles markdown content?